### DISCUSSION TITLE
Positron has problem with conda virtualenvs installed using pyenv

### DISCUSSION THREAD

**Discussion by danieltomasz** [posit-dev/positron#3913] [Feedback]
I installed an virtualenv using pyenv and pyenv-virtualenv.
I don't have a problem with using it with VSCode.
Positron is able to find this python instance but when I try to use/start it, I receive the error `Could not start runtime: failed to install ipykernel for Python 3.12.4 64-bit ('name of the virtualenv).`
`Ipykernel` is installed in this specific virtualenv but it might be not installed in the global python env.
This happens only when the virtualenv is using conda to install packages, when all the packages are installed via pip, the virtualenv seems to be correctly detected.

**Comment by danieltomasz** [comment]
The same behaviour when I try to pick the kernel in the jupyter notebook 
```Install Python package "ipykernel"?
To enable Python support, Positron needs to install the
package ipykernel for the active interpreter Python 3.12.4
at: ~/ -pyenv/versions/miniforge3-latest/envs/name of virtualenv/bin/python.
```
the path is an example path, I am managing a conda envs through `pyenv` (which work fine in the VSCode and in Jupyter Lab)
Basically this problem make Positron unusable for me (apart of R)

I am on MacOS 14.5, Positron Version: 2024.07.0 (Universal) build 15

**Reply by juliasilge** [reply]
Thank you for sharing the problem you ran into! I am extremely sympathetic because I too could never get conda envs to work. 😭 I will repoen #2504 and we'll see how we can get to the bottom of this problem.
